DependencyInjection/FOSUserExtension.php

```diff
@@ -86,10 +86,6 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setAlias('fos_user.util.token_generator', $config['service']['token_generator']);
         $container->setAlias('FOS\UserBundle\Util\TokenGeneratorInterface', new Alias('fos_user.util.token_generator', false));
```

Comment:
please also move this one to the util.xml file

Reply:
Done